### PR TITLE
Ond233 181 separate ondewo utils

### PR DESCRIPTION
**Pull request by ficinator**
Main points:
* extract `ondewo.utils` to a separate [repo](https://github.com/ondewo/ondewo-client-utils-python)
* introduced pre-commit hooks, which produces formatting diffs
* updated nlu-api with `audio` field in `session_pb2.StreamingDetectIntentResponse`
* added streaming detect intent example

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-dataclasses-json==0.5.2
+git+https://github.com/ondewo/ondewo-client-utils-python.git#egg=ondewo-client-utils
```
Comment (FergusFettes) [inline requirements.txt]: oneeeee second

Comment (FergusFettes) [reply requirements.txt]: kablamo!
https://pypi.org/project/ondewo-client-utils/

File: setup.py [hunk 2]
```diff
@@ -4,10 +4,19 @@
...
+            req_url, req_name = req.split("#egg=")
+            req_str = f"{req_name} @ {req_url}"
+        else:
+            req_str = req
+        print(req_str)
+        requires.append(req_str)
 
 setuptools.setup(
-    name="ondewo-nlu-client-python",
+    name="ondewo-nlu-client",
```
Comment (FergusFettes) [inline setup.py:10]: waaa, didnt i do that already? or did noone merge my PR

**Review by FergusFettes — CHANGES_REQUESTED** [review]
wonderful as always.

please fix requirements.txt!

File: setup.py (hunk 2 above)
Comment (ficinator) [reply setup.py:10]: I noticed the pypi version is called `ondewo-nlu-client`, but haven't seen it in history of this repo :thinking:

**Pull request merged**